en/docs/tutorials/map-data.md

```diff
@@ -0,0 +1,293 @@
+# Map data
+
+Choreo components often need to convert input data to produce an output to present the same data in a different format, structure, or both. Such a conversion is called a data-mapping.
+
+Choreo allows you to map data with ease in the Web Editor.
+
+This guide walks you through a simple use case where you will input student data in a specific structure and present it in another.
+
+Let's Start!
+
+### Step 1: Add a REST API
+
+First, let's add a new REST API for which you can define a data mapping. To do this, follow the steps given below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
+
+2. On the **Home** page, scroll to the **Get started with a template** section. On the **Echo Service** template, click **Get Started**.
```

Comment:
It will increase the step count in a section which is not the core part of the topic. Also, scrolling is not a very significant action that calls for a separate step isn't it?  What do you think?